quickstarts/java/jdbc-executebatch/config.yml

```diff
@@ -0,0 +1,38 @@
+name: jdbc-executebatch
+title: JDBC-ExecuteBatch Java Agent Extension
+description: |
+  # What is executeBatch call?
+
+  ExecuteBatch method in jdbc submits a batch of commands to the database for execution and if all commands execute successfully,returns an array of update counts. 
```

Comment:
I'd suggest cleaning up this markdown a bit and using bullets and spacing in the description.

Reply:
reformatted section to use bullets